common/src/main/java/com/microsoft/identity/common/internal/msafederation/google/SignInWithGoogleParameters.kt

```diff
@@ -25,29 +25,39 @@ package com.microsoft.identity.common.internal.msafederation.google
 import android.app.Activity
 import com.microsoft.identity.common.internal.msafederation.MsaFederatedSignInParameters
 import com.microsoft.identity.common.internal.msafederation.MsaFederatedSignInProviderName
+import com.microsoft.identity.common.internal.msafederation.MsaFederationConstants
 
 /**
  * SignInWithGoogleParameters holds the parameters required for signing in with Google.
  *
  * @property activity The Activity context used for the sign-in process.
+ * @property serverClientId The target server OAuth2.0 client (backend) ID generated in google console project.
+ *                          Authenticates backend server with Google's APIs. Default value MSA's server client id.
  * UI for the sign-in process.
  */
-data class SignInWithGoogleParameters(
-    internal val activity: Activity
+data class SignInWithGoogleParameters @JvmOverloads constructor(
+    internal val activity: Activity,
+    internal val serverClientId: String = MsaFederationConstants.GOOGLE_MSA_SERVER_CLIENT_ID
```

Comment:
Should the serverClientId value be a flighting config, so it can be changed without code change?

Reply:
This would be executed on app side not broker, so we can't.